src/lib/radio/radio.ts

```diff
@@ -516,4 +498,20 @@ export class MdRadioButton implements OnInit, AfterViewInit, OnDestroy {
     }
   }
 
+  /** Function is called whenever the focus changes for the input element. */
+  private _onInputFocusChange(focusOrigin: FocusOrigin) {
```

Comment:
Ah, I wasn't aware that it gets called for blur as well. In that case it's fine, although it could get stated in the comment.